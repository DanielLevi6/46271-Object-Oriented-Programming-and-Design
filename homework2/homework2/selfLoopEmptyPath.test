# selfLoopEmptyPath test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 3
CreateNode node2 3
CreateNode node3 3
CreateNode node4 1
CreateNode node5 1
CreateNode node6 2
CreateNode node7 4
CreateNode node8 1
CreateNode node9 2
CreateNode node10 3

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4
AddNode myGraph node5
AddNode myGraph node6
AddNode myGraph node7
AddNode myGraph node8
AddNode myGraph node9
AddNode myGraph node10

# Add an edge
AddEdge myGraph node1 node2
AddEdge myGraph node1 node3
AddEdge myGraph node1 node4
AddEdge myGraph node1 node5
AddEdge myGraph node1 node6
AddEdge myGraph node1 node7
AddEdge myGraph node1 node8
AddEdge myGraph node1 node9
AddEdge myGraph node1 node10
AddEdge myGraph node2 node3
AddEdge myGraph node2 node4
AddEdge myGraph node2 node5
AddEdge myGraph node2 node6
AddEdge myGraph node2 node7
AddEdge myGraph node2 node8
AddEdge myGraph node2 node9
AddEdge myGraph node2 node10
AddEdge myGraph node3 node4
AddEdge myGraph node3 node5
AddEdge myGraph node3 node6
AddEdge myGraph node3 node7
AddEdge myGraph node3 node8
AddEdge myGraph node3 node9
AddEdge myGraph node3 node10
AddEdge myGraph node4 node5
AddEdge myGraph node4 node6
AddEdge myGraph node4 node7
AddEdge myGraph node4 node8
AddEdge myGraph node5 node6
AddEdge myGraph node5 node7
AddEdge myGraph node6 node6
AddEdge myGraph node7 node7
AddEdge myGraph node8 node6
AddEdge myGraph node9 node7

# Find a path
FindPath myGraph node1 -> node1 