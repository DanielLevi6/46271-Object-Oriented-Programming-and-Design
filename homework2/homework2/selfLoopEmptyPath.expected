# selfLoopEmptyPath test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 3
created node node2 with cost 3
created node node3 with cost 3
created node node4 with cost 1
created node node5 with cost 1
created node node6 with cost 2
created node node7 with cost 4
created node node8 with cost 1
created node node9 with cost 2
created node node10 with cost 3

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph
added node node5 to myGraph
added node node6 to myGraph
added node node7 to myGraph
added node node8 to myGraph
added node node9 to myGraph
added node node10 to myGraph

# Add an edge
added edge from node1 to node2 in myGraph
added edge from node1 to node3 in myGraph
added edge from node1 to node4 in myGraph
added edge from node1 to node5 in myGraph
added edge from node1 to node6 in myGraph
added edge from node1 to node7 in myGraph
added edge from node1 to node8 in myGraph
added edge from node1 to node9 in myGraph
added edge from node1 to node10 in myGraph
added edge from node2 to node3 in myGraph
added edge from node2 to node4 in myGraph
added edge from node2 to node5 in myGraph
added edge from node2 to node6 in myGraph
added edge from node2 to node7 in myGraph
added edge from node2 to node8 in myGraph
added edge from node2 to node9 in myGraph
added edge from node2 to node10 in myGraph
added edge from node3 to node4 in myGraph
added edge from node3 to node5 in myGraph
added edge from node3 to node6 in myGraph
added edge from node3 to node7 in myGraph
added edge from node3 to node8 in myGraph
added edge from node3 to node9 in myGraph
added edge from node3 to node10 in myGraph
added edge from node4 to node5 in myGraph
added edge from node4 to node6 in myGraph
added edge from node4 to node7 in myGraph
added edge from node4 to node8 in myGraph
added edge from node5 to node6 in myGraph
added edge from node5 to node7 in myGraph
added edge from node6 to node6 in myGraph
added edge from node7 to node7 in myGraph
added edge from node8 to node6 in myGraph
added edge from node9 to node7 in myGraph

# Find a path
shortest path in myGraph: node1
